subprojects/gradle/runner/shared/src/main/kotlin/com/avito/runner/logging/Logger.kt

```diff
@@ -1,6 +0,0 @@
-package com.avito.runner.logging
-
-interface Logger {
-    fun log(message: String)
-    fun notify(message: String, error: Throwable? = null)
-}
```

Comment:
Replaced in favor of common Logger - `com.avito.logger.Logger`.

In the test runner we had to duplicate wrapper on `CILogger`. I don't see reasons for duplication of such common utilities.